docs/plugins/kube-prometheus.md

```diff
@@ -8,4 +8,15 @@ This plugin installs [kube-prometheus](https://github.com/prometheus-operator/ku
 --8<-- "kube-prometheus.yaml"
 ```
 
-Currently, except for `values_yaml`, all the parameters in the example above are mandatory.
+### Default Configs
+
+| key | default value | description |
+|  ----  | ----  | ----  |
+| chart.chart_name | prometheus-community/kube-prometheus-stack | chart name |
+| chart.timeout | 10m | this config will wait 10 minutes to deploy |
```

Comment:
The default value in the yaml file is 5 minutes, but it is set to 10 minutes in the code and default config.